linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/LinkerdHeaders.scala

```diff
@@ -65,15 +81,17 @@ object Headers {
      * HttpTraceInitializer.serverModule.
      */
     val serverModule: Stackable[ServiceFactory[Request, Response]] =
-      new Stack.Module1[hparam.MaxHeaderSize, ServiceFactory[Request, Response]] {
+      new Stack.Module3[hparam.MaxHeaderSize, hparam.MaxResponseSize, Headers.param.MaxErrResponseSize, ServiceFactory[Request, Response]] {
```

Comment:
I think we shouldn't bring `MaxResponseSize` into this at all.  It should be sufficient to just check the `MaxErrResponseSize`.

Reply:
done